.eslintrc.js

```diff
@@ -61,5 +63,31 @@ module.exports = {
         'func-names': 0,
       },
     },
+    {
+      files: ['babel-plugin-strip-wrappers.js'],
+      rules: {
+        'node/no-extraneous-require': [
+          'error',
+          {
+            allowModules: ['@babel/plugin-syntax-decorators'],
+          },
+        ],
+      },
+    },
+    {
+      files: ['node-tests/**/*.js'],
+      env: {
+        qunit: true,
+      },
+      rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
+        'node/no-unpublished-require': 'off',
+        'node/no-extraneous-require': [
+          'error',
+          {
+            allowModules: ['@babel/core', '@babel/plugin-syntax-decorators'],
```

Comment:
I'm not keen on allowing these to be required directly without a dep on them. Any reason we can't require the babel plugin to pass them through by way of `require.resolve`/`resolve.sync`?

Reply:
Ah, we're talking about the same thing as in [this comment](https://github.com/rwjblue/ember-test-waiters/pull/81/files#r377864336), right?